backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/MeetingService.cs

```diff
@@ -79,7 +79,7 @@ private static List<UserMeetingDTO> GetAllParticipants(Meeting meeting, int numb
                 });
 
             return slotMembers.Union(external).Take(numberOfMembers).ToList();
-        } 
+        }
         private static string CreateMemberTitle(Meeting meeting)
```

Comment:
Missing new line

Reply:
new line is still missing

Reply:
I think smth is wrong is with git displaying changes. In branch it looks good.
![image](https://user-images.githubusercontent.com/98499085/189543229-0c5762cb-3cdf-4090-b8ab-b329d6e1eba3.png)

Reply:
Fixed it.